src/components/common/Video.js

```diff
@@ -3,20 +3,23 @@ import {
   TouchableOpacity,
   TouchableWithoutFeedback,
   Image,
-  View
+  View,
+  Dimensions
 } from 'react-native';
 import { Video } from 'expo';
 import playIcon from '../../assets/images/play.png';
-const replayIcon = playIcon;
 
+const replayIcon = playIcon;
+var res = Dimensions.get('window');
```

Comment:
Please use `const` resp. `let` instead of `var` since `var` has super weird scoping rules. Since you don't change this variable, `const` is the way to go here.

Reply:
Also, you could instead do
```
const { width, height } = Dimensions.get('window');
```
and use `width` and `height` directly.

Reply:
Please move that line to the render method. Otherwise, if the window dimension changes after the first usage of `Video` (i.e. by changing from portrait to landscape), `width` and `height` are swapped. Even better: listen to changes to the dimensions and rerender the Video component. To do this, you:
* Move `width` and `height` to state (since state changes trigger rerendering)
* Add an event listener for Dimensions in a lifecycle method and update the state if the values change
  * You can review the existing lifecycle methods here: https://facebook.github.io/react/docs/react-component.html
  * In this case, `componentDidMount` would probably be the right way to do it
* Remove your event listener for Dimensions when your component gets unmounted, i.e. by using `componentWillUnmount`

Reply:
Thanks for your detailed description. My next commit shall comprise all above suggestions.